src/main/java/edu/harvard/iq/dataverse/FileDownloadServiceBean.java

```diff
@@ -386,6 +386,16 @@ public void downloadDatasetCitationXML(Dataset dataset) {
         downloadCitationXML(null, dataset, false);
     }
 
+    public void downloadDatasetCitationXML(DatasetVersion version) {
```

Comment:
As far as I can tell, it looks like this method is now used on place of the only prior use of downloadDatasetCitationXML - can that method be deleted? Further, since that method was the only one calling downloadCitationXML with a non-null dataset, can we remove that parameter and associated logic?

Same questions for the other RIS/Bibtex formats - can we remove the method/param that isn't used anymore?

Reply:
Looks good - thanks.